about.html

```diff
@@ -0,0 +1,72 @@
+</head>
+<body>
+    <div class="topbar">        
+        <span class="facebook"></span>
+        <span class="twitter"></span>
+        <a href="about.html">Spanish</a>
+        <a href="#program">My Page</a>
+        <a href="#footer">Logout</a> 
+    </div>
+    <div id="bg-about">
+        <nav class="navbar">                        
+           <span class="material-symbols-outlined">menu</span>           
+            <div class="home"><a href="index.html"></a></div>          
+            <div id="links">
+                <a href="index.html">Home</a>
+                <a href="#logo">Content</a>
+                <a href="#footer">Platforms-Studios</a>
+            </div>       
+       </nav>
+       <header class="header about"> 
+            <h3>"Twinsen Odyssey's is BACK!"</h3>
+            <h1>Little Big Adventure</h1>
+            <p id="sup_text">The gameplay is partially free-roaming, allowing the player free non-linear travel around the islands of the game's world once they 
+                have been unlocked by completing certain objectives or solving certain puzzles; there are also many tasks which are optional or non-linear.</p> 
+            <p id="sup_text_desk">The gameplay is partially free-roaming, allowing the player free non-linear travel around the islands of the game's world once
+                 they have been unlocked by completing certain objectives or solving certain puzzles; there are also many tasks which are optional or non-linear. 
+                 Little Big Adventure 2 sold above 300,000 units by August 1999. GameSpot reported, "In France, Twinsen is big. In the States, he's an obscure oddity."
+                 "With its unparalleled sense of marvel and imagination, engaging story, and deft mix of action and adventure, Twinsen's Odyssey is a true delight and 
+                 simply the most charming game I've ever played".</p>      
+            <p id="suptext2">Please contact us per Email for any further questions
+            about LIB Twinsen's Odyssey new Saga!<br>
+            <a href=""> davidisea@hotmail.com</a></p>
+        </header>
+    </div>
+    <div class="logo" id="logo">
+        <h4>Little Big Adventure Cover</h4>
+        <p>The cover of LBA Twinsen's Odyssey was decided<br>
+            through the activision designers at 1993.</p>
+        <img src="Images/lba2.jpg" alt="LBA Cover">
+    </div>
+    <div class="past">
+        <h4>See the past LBA Adventures</h4>
+        <p>Take a look to the original 1st & 2nd game of<br> the saga from the 90s.</p>
+        <div class="oldcont">
+        <div class="old1">
+            <h3>1994</h3>
+            <p>Little Big Adventure</p>
+        </div>
+        <div class="old2">
+            <h3>1997</h3>
+            <p>Little Big Adventure 2: Twinsen Odyssey</p>
+        </div>
+        </div>
+    </div>
+    <footer id="footer">
+        <div class="partners">
+        <h4>Platforms-Studios</h4>
+        </div>        
```

Comment:
- Kindly replace this content with the contents in your main footer, keeping in mind that the index and about page should have the same header and footer.
- After updating, ensure it has the right background.

Reply:
Ok, what I did is remove the Copyrights `div` out of the footer, because the about page from the template doesn't have it. This way both have same footer.

Reply:
- [ OPTIONAL ] I think you need to add the two sections footer in the about page the partner section and the copyrights
and you don't need to add nothing in the home page for the mobile version


- For the desktop version you acutally need to add both footer sections to the home page but you just need to add the copyright section in the about but in different background color so kindly fix that ✅✅✅